vllm/model_executor/layers/fused_moe/fused_moe.py

```diff
@@ -1084,7 +1112,7 @@ def fused_experts_impl(hidden_states: torch.Tensor,
                     out_hidden_states[begin_chunk_idx:end_chunk_idx])
     return out_hidden_states
 
-
+# Deepseek V1
```

Comment:
Better to enhance the function docstring instead of adding a standalone comment.